### PR TITLE
update: add functions

### PR DESCRIPTION
**Pull request by IloveJavaa**
Asana - https://app.asana.com/0/0/1205371519209335/f

### CODE REVIEW

File: src/android/TrackierCordovaPlugin.java [hunk 1]
```diff
@@ -12,105 +12,171 @@
...
+            } else if (action.equals("setUserName")) {
+                return setUserName(TrackierCordovaUtil.optString(args, 0));
+            } else if (action.equals("setUserPhone")) {
+                return setUserPhone(TrackierCordovaUtil.optString(args, 0));
+            } else if (action.equals("getTrackierId")) {
+                String trackierId = com.trackier.sdk.TrackierSDK.getTrackierId();
+                callbackContext.success(trackierId);
+                return true;
+            } else if (action.equals("getAd")) {
+                String data = "sanu";// com.trackier.sdk.TrackierSDK.getTrackierId();
```
Comment (Hemant-Mann) [inline src/android/TrackierCordovaPlugin.java]: fix this

File: src/android/TrackierCordovaPlugin.java [hunk 2]
```diff
@@ -12,105 +12,171 @@
...
+                callbackContext.success(data);
+                return true;
+            } else if (action.equals("getPid")) {
+                String data = com.trackier.sdk.TrackierSDK.getPid();
+                callbackContext.success(data);
+                return true;
+            } else if (action.equals("getIsRetargeting")) {
+                String data = com.trackier.sdk.TrackierSDK.getIsRetargeting();
+                callbackContext.success(data);
+                return true;
```
Comment (Hemant-Mann) [inline src/android/TrackierCordovaPlugin.java]: return true

should be moved out of the if else if @IloveJavaa

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: src/android/TrackierCordovaPlugin.java (hunk 1 above)
Comment (IloveJavaa) [reply src/android/TrackierCordovaPlugin.java]: done

File: src/android/TrackierCordovaPlugin.java (hunk 2 above)
Comment (IloveJavaa) [reply src/android/TrackierCordovaPlugin.java]: done

**Review by Hemant-Mann — APPROVED** [review]

**Pull request merged**